tests/test_formatter.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime, timedelta
+from random import randrange
+
+from todoman import ui
+
+DATE_FORMAT = "%d-%m-%y"
+
+
+def test_format_date():
+    """
+    Tests the format_date function in todoman.ui.TodoFormatter
+    """
+    formatter = ui.TodoFormatter(DATE_FORMAT)
+    today = datetime.now()
+    tomorrow = today + timedelta(days = 1)
+    any_day = today + timedelta(days = randrange(2, 8))
```

Comment:
Back when I used sublime I used a plugin called `Anaconda` that did some python style checks and stuff (don't recall what else). Might be worth looking into.